src/saltext/vmware/states/esxi.py

```diff
@@ -964,6 +1023,8 @@ def firewall_config(
     cluster_name=None,
     host_name=None,
     service_instance=None,
+    profile=None,
+    less=False
```

Comment:
**suggestion(non-blocking)** based on our discussion, does advanced config need the `less` parameter as well?

Reply:
other alternatives - can we get stuff in `__opts__` (e.g. test=True), or potentially add a different output module?

Reply:
We tried to add in __opts__ just by using regular keyword cli argument, but only a set of built-in arguments are valid, the one I found closest to verbosity is log_level, but not sure if we can use this one. For the moment, advanced_config uses less parameter as well